src/jobs/jobs.controller.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { Test, TestingModule } from '@nestjs/testing'
```

Comment:
Should we keep these?  we can remove them since we plan to test via postman

Reply:
I took care of these in my last PR... pull from `develop`

https://github.com/thegoodparty/gp-api/pull/5